tutorials/using-bucket-website-with-mkdocs/index.mdx

```diff
@@ -0,0 +1,380 @@
+---
+meta:
+  title: Creating a documentation website using Object Storage bucket website and Material for MkDocs
+  description: Create a sleek documentation website using the Scaleway Object Storage bucket website feature and Material for MkDocs. Simple, fast, and easy to complete!
+content:
+  h1: Creating a documentation website using Object Storage bucket website and Material for MkDocs
+  paragraph: Create a sleek documentation website using the Scaleway Object Storage bucket website feature and Material for MkDocs. Simple, fast, and easy to complete!
+categories:
+  - object-storage
+tags: documentation-website bucket-website material-for-mkdocs object-storage website
+dates:
+  validation: 2024-10-07
+  posted: 2024-10-07
+---
+
+This tutorial shows you how to create a documentation website using the Scaleway [Object Storage bucket website](/storage/object/concepts/#bucket-website) feature with MkDocs.
+
+[MkDocs](https://www.mkdocs.org/) is a static site generator that allows users to generate a static website from a set of Markdown files. It is written in Python and uses Markdown for content creation.
+We will use MkDocs to generate our website, and Material for MkDocs which is a documentation framework that extends MkDocs with additional features and customization options, to create visually appealing and user-friendly documentation websites.
+
+Find out how to Set up MkDocs and configure your website in the first tab, then click the second tab to learn how to get a preview of your website and how to deploy it.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- Installed Python
+- Installed pip
+- Installed Visual Studio Code
+- Knowledge of the [Markdown](https://www.markdownguide.org/getting-started/#what-is-markdown) markup language
+
+<Tabs id="install">
```

Comment:
You can't have titles within tabs, the anchors won't work. I suggest having a "## Set up MkDocs and configure your website" section, then a "## Preview and deploy your MkDocs website" section

Reply:
Removed tabs!